packages/react-dom/src/__tests__/ReactDOMServerIntegrationForwardRef-test.js

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
Can we combine these two files into `IntegrationSpecialTypes` or something? We don't expect to add many more tests to it, and each test file has some overhead.

Reply:
Sure.